cmd/postgres_exporter/postgres_exporter_test.go

```diff
@@ -159,7 +159,7 @@ func (s *FunctionalSuite) TestEnvironmentSettingWithDnsAndSecrets(c *C) {
 // test DSN including SSL enabled
 func (s *FunctionalSuite) TestSSL(c *C) {
 	// Thr driver doesn't support sslmode=prefer
-	envDsn := "postgresql://root:root@localhost:5433/?sslmode=require&sslcert=../../docker/server.crt&sslkey=../../docker/server.key"
+	envDsn := "postgresql://root:root@localhost:5433/?sslmode=require"
```

Comment:
`sslcert` and `sslkey` are used for client certificates: https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-CLIENTCERT We do not use them in 2.0: https://jira.percona.com/browse/PMM-1093
In pmm-managed, we will use `sslmode=require` for `tls_skip_verify` mode, and `sslmode=verify-full` for normal mode.
While we might test `verify-full` mode there using https://github.com/FiloSottile/mkcert or something similar, I don't think we should do it now – it will take too much time.

Reply:
this is just a test to check the connection is using ssl, it doesn't mean we are going to include those parameters but if you want I can remove it